tests/test_fixtures/rule_based_profiler/README.md

```diff
@@ -0,0 +1,9 @@
+# What's in this folder?
+
+This folder contains Rule-based Profiler configurations that are used for integration testing. 
+
+Each profiler is meant to be used on the NYC taxi data. 
+
+- Alice : BatchRequest yielding exactly one batch
+- Bobby : BatchRequest yielding two batches (January, 2019 and February, 2019 trip data)
+- Bobster : BatchRequest yielding  thirty five (35) batches (January, 2018 -- November, 2020 trip data)
```

Comment:
I've added a short readme for the RBP configurations for Alice, Bobby and Bobster. Please feel to suggest additions or changes.